_pages/en_US/legacy-methods.txt

```diff
@@ -38,9 +47,11 @@ If both DS Internet Settings and DS Download Play fail to work, you will have to
 
 ___
 
-1. [BB3-USM](installing-boot9strap-(usm)): Seedminer + BannerBomb3 + unSAFE_MODE
-  + This method requires working shoulder buttons
-1. [BannerBomb3](bannerbomb3): Seedminer + BannerBomb3 + Fredtool
+1. [unSAFE_MODE](installing-boot9strap-(usm)): Seedminer + BannerBomb3 + unSAFE_MODE
+  + This method should only be used if you have a broken Internet Browser (Requires working shoulder buttons)
+1. [Fredtool](bannerbomb3): Seedminer + BannerBomb3 + Fredtool
   + This method is recommended if your shoulder buttons do not work
+1. [Pichaxx](homebrew-launcher-(pichaxx)): Seedminer + Pichaxx + HBL-Slottool
+  + This method is recommended if your DSiWare Management menu crashes and you have a broken browser
 1. [Pichaxx](homebrew-launcher-(pichaxx)): Seedminer + Pichaxx + Frogtool
-  + This method is recommended if your DSiWare Management menu crashes
\ No newline at end of file
+  + This method is recommended if your DSiWare Management menu crashes and you have broken shoulder buttons
```

Comment:
This many options will inevitably confuse people

Reply:
I partially thought that too but the problem I had was that I had no other feasible way I could easily do that. Linking directly to the Frogtool page would cause issues so I didn't try that. The only thing I could think of that could be better would be to make the entire PicHaxx thing one method and then lead to branching out methods on that page (which to be fair I basically already do in this PR) but then how would I exactly list it off on the legacy-methods page?

Reply:
This only shows why it's bad to have this many confusing branches (can't do browserhax -> PICK 1/3: BB3-USM, BB3-dsidump, pichaxx; pichaxx -> PICK 1/2: pichaxx-usm, pichaxx-frogtool).

There comes a point where you have to sacrifice 'the fastest method' for what will make things more linear.

Reply:
I partially resolved this (mainly the two PicHaxx things).

The main thing I still ask is how else should it be altered to specifically to fix the confusion issue?